.gitlab/internal_kubernetes_deploy.yml

```diff
@@ -4,6 +4,8 @@
 
 internal_kubernetes_deploy_experimental:
   stage: internal_kubernetes_deploy
+  artifacts:
+    untracked: true
```

Comment:
I think it would be better to add specific files to artifacts, rather than all untracked files.

Reply:
we won't need this once it all runs on the `slack-notifier` image, this was just a solution to pass data between the different images